server/src/main/java/org/opensearch/cluster/coordination/ClusterBootstrapService.java

```diff
@@ -105,10 +114,10 @@ public ClusterBootstrapService(
         Consumer<VotingConfiguration> votingConfigurationConsumer
     ) {
         if (DiscoveryModule.isSingleNodeDiscovery(settings)) {
-            if (INITIAL_MASTER_NODES_SETTING.exists(settings)) {
+            if (INITIAL_CLUSTER_MANAGER_NODES_SETTING.exists(settings)) {
```

Comment:
This is for my understanding - I noticed that we're not branching on version here. I assume its safe to make these in-line changes because the _cluster-manager_ settings internally falls back to the deprecated _master_ setting (as illustrated in the test file). Is that correct?

Reply:
@kartg You are so careful. I realized that my change in this line is not correct. 😲
The `exists()` method excluded fallback setting, which is mentioned explicitly (
https://github.com/opensearch-project/OpenSearch/blob/1.2.4/server/src/main/java/org/opensearch/common/settings/Setting.java#L427). I will add the 2 settings and connect with `||` in the `if` condition, and add unit test. Thank you!

Reply:
Resolved in commit https://github.com/opensearch-project/OpenSearch/pull/2463/commits/03e29f8fbde874b4de1c308e1751d6a737bff75e

Reply:
It seems like you could replace the `||` condition with the `existsOrFallbackExists` API, [which does the same thing](https://github.com/opensearch-project/OpenSearch/blob/1.2.4/server/src/main/java/org/opensearch/common/settings/Setting.java#L450).

Reply:
Wow, there is even a method did that.. 😂 I will check it and update.